webpack/JobWizard/steps/Schedule/index.js

```diff
@@ -0,0 +1,89 @@
+import React, { useState } from 'react';
+import {
+  Title,
+  TextInput,
+  Grid,
+  GridItem,
+  Button,
+  Form,
+  FormGroup,
+} from '@patternfly/react-core';
+import { translate as __ } from 'foremanReact/common/I18n';
+import { SelectField } from '../form/SelectField';
+import { ScheduleType } from './ScheduleType';
+import { QueryType } from './QueryType';
+import { StartEndDates } from './StartEndDates';
+
+const repeatTypes = {
+  noRepeat: __('Does not repeat'),
+  cronline: __('Cronline'),
+  monthly: __('Monthly'),
+  weekly: __('Weekly'),
+  daily: __('Daily'),
+  hourly: __('Hourly'),
+};
+
+const Schedule = () => {
+  const [repeatType, setRepeatType] = useState(repeatTypes.noRepeat);
+  const [repeatAmount, setRepeatAmount] = useState('');
+  const [starts, setStarts] = useState('');
+  const [ends, setEnds] = useState('');
+
+  const [repeatValidated, setRepeatValidated] = useState('default');
+  const handleRepeatInputChange = newValue => {
+    setRepeatValidated(newValue >= 1 ? 'default' : 'error');
+    setRepeatAmount(newValue);
+  };
+  return (
+    <Form>
+      <Title headingLevel="h2">{__('Schedule')}</Title>
+      <ScheduleType />
+      <Grid>
+        <GridItem span={6}>
+          <SelectField
+            fieldId="repeat-select"
+            options={Object.values(repeatTypes)}
+            setValue={newValue => {
+              setRepeatType(newValue);
+              if (newValue === repeatTypes.noRepeat) {
+                setRepeatValidated('default');
+              }
+            }}
+            value={repeatType}
+          />
+        </GridItem>
+        <GridItem span={1} />
+        <GridItem span={5}>
+          <FormGroup
+            isInline
+            helperTextInvalid={__(
+              'Repeat amount can only be a positive number'
+            )}
+            validated={repeatValidated}
+          >
+            <TextInput
+              isDisabled={repeatType === repeatTypes.noRepeat}
+              id="repeat-amount"
+              value={repeatAmount}
+              type="number"
+              onChange={newValue => handleRepeatInputChange(newValue)}
+              placeholder={__('Repeat N times')}
+            />
```

Comment:
Need to check the firefox issue here (no Onchange fired when input type is number)